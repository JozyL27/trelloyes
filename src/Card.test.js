import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Card from './Card';

describe('<Card />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        // Pass required props in the usual way
        ReactDOM.render(
            <Card
            title="Joe"
            content="hey joe"
            />,
            div
        );

        ReactDOM.unmountComponentAtNode(div);
    });

    // it('renders this UI as expected', () => {
    //     // Render the component, as JSON
    //     const tree = renderer.create(<Card
    //         title="joe"
    //         content="hey joe"
    //         />).toJSON();
    //     // Check whether it matches the previous snapshot
    //     // Stored in __snapshots__/App.test.js.snap
    //     expect(tree).toMatchSnapshot(); 
    // });
});

describe('<Card />', () => {
    it('renders this UI as expected', () => {
        // Render the component, as JSON
        const tree = renderer.create(<Card
            title="joe"
            content="hey joe">
            </Card>).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});