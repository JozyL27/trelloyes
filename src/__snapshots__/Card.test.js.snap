// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders this UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    joe
  </h3>
  <p>
    hey joe
  </p>
</div>
`;
